shops/forms.py

```diff
@@ -3,33 +3,25 @@
 from django.utils import timezone
 
 
-def get_cities():
-    return [
-        (city.id, city.name) 
-        for city in City.objects.all()
-        ]
+class CityForm(forms.ModelForm):
+    class Meta:
+        model = City
+        fields = ['name']
 
-class CityForm(forms.Form):
-    name = forms.CharField(label='City name', max_length=100)
 
+class ShopForm(forms.ModelForm):
+    class Meta:
+        model = Shop
+        fields = ['city', 'name', 'code', 'address', 'postcode', 'year_opened']
 
-class ShopForm(forms.Form):
-    city = forms.ModelChoiceField(label='City', queryset=City.objects.all())
-    name = forms.CharField(label='Shop name', max_length=100)
-    code = forms.CharField(label='Code', max_length=5)
-    address = forms.CharField(label='Address', max_length=100)
-    postcode = forms.CharField(label='Postcode', max_length=100)
-    year_opened = forms.IntegerField(label='Year opened')
 
-
-    # Overwriting the parents is_valid function
     def is_valid(self):
         if not super(ShopForm, self).is_valid():
             return False
 
-        year_opened = self.cleaned_data['year_opened']
         if not (1850 < self.cleaned_data['year_opened'] < timezone.now().year):
```

Comment:
Did you remove the validation and not replace it? If so, please figure out how to put it back in - it can be done (fairly easily) with a ModelForm